config/ext/rails.yml

```diff
@@ -56,15 +60,33 @@ Rails/ActiveSupportAliases:
   Enabled: false
   VersionAdded: '0.48'
 
+Rails/ApplicationController:
+  Description: 'Check that controllers subclass ApplicationController.'
+  Enabled: true
```

Comment:
I'm actually not sure about this. We occasionally have controllers that inherit just from `ActionController::Base`, e.g. when we run Jasmine tests at `/jasmine` as part of our integration tests.

Would say we don't yet enable it, but create a separate issue to discuss.